### PR TITLE
New package: komorebi-2.2.1

### PR DESCRIPTION
**Pull request by kawaiiamber**
Upstream says they haven't had time to fix CMakeLists.txt, so I added a patch. I'll make PR upstream as well later to fix it.

### CODE REVIEW

**Comment by FollieHiyuki** [comment]
The original repo doesn't get developed anymore.
Maybe switch to this fork: https://github.com/Komorebi-Fork/komorebi

**Comment by kawaiiamber** [comment]
Does void generally allow forks? Also, this app might be considered "complete" in the sense that no more updating is needed to do what it's purpose is. That fork uses `meson`, not gonna lie, I've never heard of meson. I might try patching this template, but this fork idea seems interesting. What do you think? (Thanks for the idea, too :) )

**Comment by kawaiiamber** [comment]
Thanks @FollieHiyuki for the fork link! I think this one will be easier to maintain. I also noticed this is the one that the AUR uses.

File: srcpkgs/komorebi/template [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+short_desc="Beautiful and customizable wallpapers manager for Linux"
+maintainer="KawaiiAmber <japaneselearning101@gmail.com>"
+license="GPL-3.0-or-later"
+homepage="https://github.com/Komorebi-Fork/komorebi"
+distfiles="${homepage}/archive/v${version}.tar.gz"
+checksum=4074751825cf031cca63c647372494446dab2492e8311ff439785527b3518c5b
+
+if [ "${CROSS_BUILD}" ]; then
+	hostmakedepends+=" vala glib-devel"
+fi
```
Comment (ericonr) [inline srcpkgs/komorebi/template:17]: This should be in hostmakedeps unconditionally, the fact that it was available for native builds is an accident in that `vala-devel` depends on `vala`, not by design. Same for `glib-devel`.

File: srcpkgs/komorebi/template [hunk 2]
```diff
@@ -0,0 +1,17 @@
+# Template file for 'komorebi'
+pkgname=komorebi
+version=2.2.1
+revision=1
+build_style=meson
+hostmakedepends="pkg-config"
+makedepends="vala-devel gtk+3-devel libgee08-devel clutter-gtk-devel clutter-devel webkit2gtk-devel clutter-gst3-devel glib-devel"
```
Comment (ericonr) [inline srcpkgs/komorebi/template:7]: Break at ~80 columns

File: srcpkgs/komorebi/template (hunk 1 above)
Comment (kawaiiamber) [reply srcpkgs/komorebi/template]: Ah, I see. Thanks for the heads up! Added to `hostmakedepends` unconditionally!

File: srcpkgs/komorebi/template (hunk 2 above)
Comment (kawaiiamber) [reply srcpkgs/komorebi/template]: Ah, ok - done

**Comment by kawaiiamber** [comment]
The package works well. Doing some further testing - videos do not work. However, this is not the fault of this package, the `webkit2gtk` (and most likely `gstreamer1`) package is broken: see [here](https://github.com/void-linux/void-packages/issues/25852).

**Pull request closed without merging**